dev-docs/bidders/pgamssp.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: PGAMSSP
+description: Prebid PGAMSSP Bidder Adapter
+biddercode: pgamssp
+usp_supported: true
+tcfeu_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+media_types: banner, video, native
```

Comment:
If you support all media types we highly recommend adding 

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
safeframes_ok: true/false
```

to your adapter documentation